### ISSUE TITLE
Is your PyTorch Implementation working?

### ISSUE THREAD

**Issue by pritishyuvraj** [edchengg/Voice_Converter_CycleGAN#4]
I read your following comment: https://github.com/leimao/Voice_Converter_CycleGAN/issues/2#issuecomment-397930743

I am having exactly similar issue Converting Tensorflow Model to PyTorch for CylceLoss GAN for Voice Conversion. Discriminator goes to 0 and Generator goes to 1. 

I have even matched the total number of trainable parameters from PyTorch Implementation to Tensorflow Implementation. Everything seems comparable for both models. 

Let me know if you were able to successfully train your model in PyTorch (Cycle Loss should be decreasing from  ~1.8 in iteration:0 to around ~.5 after 100K iterations)
